torchmdnet/priors/zbl.py

```diff
@@ -22,7 +22,7 @@ def __init__(self, cutoff_distance, max_num_neighbors, atomic_number=None, dista
             distance_scale = dataset.distance_scale
         if energy_scale is None:
             energy_scale = dataset.energy_scale
-        atomic_number = torch.as_tensor(atomic_number, dtype=torch.int8)
+        atomic_number = torch.as_tensor(atomic_number, dtype=torch.long)
```

Comment:
is there a reason we need to store atomic numbers as int64? Seems excessive

Reply:
Using int8 led to exceptions on the GPU.  This is just a single array whose length equals the number of atom types (generally less than 100).  The storage is negligible.